### PR TITLE
Include me as viewer

### PR DESCRIPTION
**Pull request by nrchakradhar**
To debug e2e tests

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/internal-acls/174)

### CODE REVIEW

File: ci-viewer.members.txt [hunk 1]
```diff
@@ -5,3 +5,4 @@ hejinchi@cn.ibm.com
 holden.karau@gmail.com
 iman.tabrizian@gmail.com
 redhat-team@kubeflow.org
+nrchakradhar379@gmail.com
```
Comment (jlewi) [inline ci-viewer.members.txt]: alphabetical order please

Comment (nrchakradhar) [reply ci-viewer.members.txt]: Done. Sorry. I thought it was chronological.

**Pull request merged**